### ISSUE TITLE
Xpath2.0 tokenize() function works on Windows but fails on Linux

### ISSUE THREAD

**Issue by abeykoon** [wso2/product-ei#656]
Affected version: EI-6.1.1

With Xpath2.0 because  tokenize() function is used in the scenario.

The following test proxy is normally working on Windows, but failing when deployed to Linux environment which contains an equal EI6.1.1 installation and configuration.

**TestProxy**
```
<?xml version="1.0" encoding="UTF-8"?>
<proxy xmlns="http://ws.apache.org/ns/synapse"
name="XpathTestProxy"
startOnLoad="true"
statistics="disable"
trace="disable"
transports="http">
<target>
<inSequence>
<property name="EncodedCorrelationId" value="0537522332_20170602_0406469194_2060208391_380" scope="synapse" type="STRING"/> 
<log level="custom">
<property name="=== before ===" expression="$ctx:EncodedCorrelationId" scope="synapse" type="STRING"/>
</log> 
<property expression="fn:substring-before($ctx:EncodedCorrelationId,concat('',fn:tokenize($ctx:EncodedCorrelationId,'')[last()]))" name="CorrelatieIdBijlage" scope="synapse" type="STRING" 
xmlns:fn="http://www.w3.org/2005/xpath-functions"/>
<log level="custom">
<property name="=== after ===" expression="$ctx:CorrelatieIdBijlage" scope="synapse" type="STRING"/>
</log>
</inSequence>
<outSequence/>
</target>
</proxy>
```

**Comment by Jean-Marc-Henry** [comment]
Hello 
i had the same problem i fix it by removing:
./wso2ei-6.1.1/wso2/lib/endorsed/xalan-2.7.2.wso2v2.jar
and let 
./wso2ei-6.1.1/wso2/lib/endorsed/saxon9he.jar
do the job, the both library are in the endorsed lib.
I hope it help you.
sincerely yours 
Jean-Marc Henry

**Comment by uleming** [comment]
There is line synapse.properties 
# Uncomment following to support fallback XPATH 2.0 support with DOM and Saxon
#synapse.xpath.dom.failover.enabled=true

**Comment by vinok88** [comment]
This is fixed in https://github.com/wso2/product-ei/pull/712. In fact, the synapse.properties entry
`synapse.xpath.dom.failover.enabled=true`
should be available to use XPATh 2.0 functions as @uleming mentioned.
